rest/server_context.go

```diff
@@ -483,9 +483,15 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(ctx context.Context, config
 		}
 
 		for scopeName, scopeConfig := range config.Scopes {
+			var dataStore sgbucket.DataStore
 			for collectionName, _ := range scopeConfig.Collections {
```

Comment:
Scoping this outside of the loop feels error-prone and unnessesary. It would allow someone to use `dataStore` on the last iterated collection outside of the loop.

```suggestion
			for collectionName, _ := range scopeConfig.Collections {
				var dataStore sgbucket.DataStore
```